src/os/bluestore/BlueFS.cc

```diff
@@ -218,7 +218,7 @@ int BlueFS::reclaim_blocks(unsigned id, uint64_t want,
 
   int64_t got = alloc[id]->allocate(want, cct->_conf->bluefs_alloc_size, 0,
 				    extents);
-  assert(got != 0);
```

Comment:
this is missing the update to use the shared_alloc_size in the line above, like the luminous backport has. otherwise looks good

Reply:
Thanks Josh. Done. thanks @neha-ojha for helping with squash thing.